tests/qchem/of_tests/test_convert.py

```diff
@@ -402,19 +403,11 @@ def test_operation_conversion(pl_op, of_op, wire_order):
 
     converted_of_op = qml.qchem.convert._openfermion_to_pennylane(of_op)
     _, converted_of_op_terms = converted_of_op
-    assert all(isinstance(term, pauli_ops_and_tensor) for term in converted_of_op_terms)
-    assert np.allclose(
-        qml.matrix(qml.dot(*pl_op), wire_order=wire_order),
-        qml.matrix(qml.dot(*converted_of_op), wire_order=wire_order),
-    )
```

Comment:
Please double-check to make sure it is covered properly elsewhere.

Reply:
Yep it is still there down below this [line](https://github.com/PennyLaneAI/pennylane/pull/5358/files#diff-b6bad9bf943f0151a4876dc64feb0eee801de620a9bba2842262b5588eb1fa08R412).